src/components/schema-ui/schema-menu.js

```diff
@@ -30,10 +38,11 @@ const SchemaMenu = ({ title, forPath, load, back, next }) => {
           <li key={`relationship-${index}`}>
             <button
               className="link--next"
-              onClick={() => loadNext(relationship.name)}
+              onClick={handleClick(name)}
```

Comment:
If `name` is an argument this gets called on render, you'll want to change it to something like
```suggestion
              onClick={() => handleClick(name)}
```

Reply:
`handleClick` is a curried function:

```
  const handleClick = name => e => {
    if (e.target.classList.contains('link__toggle')) {
      return;
    }
    loadNext(name);
  };
```

`name => e => { // work }` so it calling it on render to create the event handler is what I wanted :) since what gets passed as the `onClick` is `e => {}`

Reply:
That doesn't work :) 
```suggestion
              onClick={handleClick}
              name={relationship.name}
```

and you can change the `handleClick` to 

```js
  const handleClick = e => {
    if (e.target.classList.contains('link__toggle')) {
      return;
    }
    loadNext(e.target.name);
  };

Reply:
Can you check again since: https://github.com/zrpnr/jsonapi_explorer/pull/50/commits/7927ca4e782e614f9be738251fa45e4db3d3f41e

Reply:
I meant my first suggestion didn't work, I posted a revision but your comment got in the middle :) Tested this new change using `relationship.name` and it works on my local as well.